app/src/main/java/org/p2p/wallet/striga/signup/ui/StrigaSignUpSecondStepFragment.kt

```diff
@@ -100,6 +102,9 @@ class StrigaSignUpSecondStepFragment :
         replaceFragment(
             SmsInputFactory.create(
                 type = SmsInputFactory.Type.Striga,
+                args = bundleOf(
+                    StrigaSmsInputFragment.ARG_RESEND_SMS_ON_INIT to false
+                ),
```

Comment:
do not send sms right after user creation, striga already do this for us without calling `resend-sms`